templates/default/php-handler.conf.erb

```diff
@@ -3,7 +3,11 @@
   AddHandler php5-fcgi .php
   Action php5-fcgi /php5-fcgi
   Alias /php5-fcgi /var/run/php5-fcgi
+<%   if node['rackspace_apache_php']['php_handler'].fetch('pass_auth_header',false) %>
```

Comment:
I think we should use our own template if we need to change anything here =>  https://github.com/rackspace-cookbooks/rackspace_apache_php/blob/master/recipes/default.rb#L27